### PR TITLE
Add drag and drop to overview doc

### PR DESCRIPTION
**Pull request by nishitha-burman**
Add drag and drop to overview doc

AB#43380871

### CODE REVIEW

File: microsoft-edge/webview2/concepts/overview-features-apis.md [hunk 1]
```diff
@@ -1482,6 +1482,33 @@ Spatial input (mouse, touch, pen) is received by the application and must be sen
 
 ---
 
+
+<!-- ------------------------------ -->
+#### Drag and drop
+
+Dragging from WebView2 to another application is supported by default. However, dropping into WebView2 (Which includes drag/drop operations within WebView2 itself) requires forwarding IDropTarget events the host application receives from the system to WebView2 using the APIs below.
```
[the remark this answers is not in the document]
Comment (mikehoffms) [reply microsoft-edge/webview2/concepts/overview-features-apis.md:1489]: Considering a flow more linear like: 
"When the host app receives event from system, host app must forward event to the wv2 control."

File: microsoft-edge/webview2/concepts/overview-features-apis.md [hunk 2]
```diff
@@ -1482,6 +1482,33 @@ Spatial input (mouse, touch, pen) is received by the application and must be sen
...
+
+##### [.NET/C#](#tab/dotnetcsharp)
+
+* [CoreWebView2CompositionController.DragLeave Method](/dotnet/api/microsoft.web.webview2.core.corewebview2compositioncontroller.dragleave?view=webview2-dotnet-1.0.1370.28&preserve-view=true#microsoft-web-webview2-core-corewebview2compositioncontroller-dragleave)
```
Comment (mikehoffms) [inline microsoft-edge/webview2/concepts/overview-features-apis.md:1493]:
```suggestion
* [CoreWebView2CompositionController.DragLeave Method](/dotnet/api/microsoft.web.webview2.core.corewebview2compositioncontroller.dragleave)
```

Comment (mikehoffms) [reply microsoft-edge/webview2/concepts/overview-features-apis.md:1493]: test:
prefix + /dotnet/api/microsoft.web.webview2.core.corewebview2compositioncontroller.dragleave
=
learn.microsoft.com/dotnet/api/microsoft.web.webview2.core.corewebview2compositioncontroller.dragleave
Success: You end up at
https://learn.microsoft.com/en-us/dotnet/api/microsoft.web.webview2.core.corewebview2compositioncontroller.dragleave?view=webview2-dotnet-1.0.1587.40

File: microsoft-edge/webview2/concepts/overview-features-apis.md [hunk 3]
```diff
@@ -1482,6 +1482,33 @@ Spatial input (mouse, touch, pen) is received by the application and must be sen
...
+
+##### [WinRT/C#](#tab/winrtcsharp)
+
+* [ICoreWebView2CompositionControllerInterop2.DragEnter Method](/microsoft-edge/webview2/reference/winrt/interop/icorewebview2compositioncontrollerinterop2?view=webview2-winrt-1.0.1370.28&preserve-view=true#dragenter)
+* [ICoreWebView2CompositionControllerInterop2.DragLeave Method](/microsoft-edge/webview2/reference/winrt/interop/icorewebview2compositioncontrollerinterop2?view=webview2-winrt-1.0.1370.28&preserve-view=true#dragleave)
+* [ICoreWebView2CompositionControllerInterop2.DragOver Method](/microsoft-edge/webview2/reference/winrt/interop/icorewebview2compositioncontrollerinterop2?view=webview2-winrt-1.0.1370.28&preserve-view=true#dragover)
+* [ICoreWebView2CompositionControllerInterop2.Drop Method](/microsoft-edge/webview2/reference/winrt/interop/icorewebview2compositioncontrollerinterop2?view=webview2-winrt-1.0.1370.28&preserve-view=true#drop)
+* [CoreWebView2CompositionController.DragLeave Method](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2compositioncontroller?view=webview2-winrt-1.0.1370.28&preserve-view=true#dragleave)
```
Comment (mikehoffms) [inline microsoft-edge/webview2/concepts/overview-features-apis.md:1501]:
```suggestion
* [CoreWebView2CompositionController.DragLeave Method](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2compositioncontroller#dragleave)
* [ICoreWebView2CompositionControllerInterop2.DragEnter Method](/microsoft-edge/webview2/reference/winrt/interop/icorewebview2compositioncontrollerinterop2#dragenter)
* [ICoreWebView2CompositionControllerInterop2.DragLeave Method](/microsoft-edge/webview2/reference/winrt/interop/icorewebview2compositioncontrollerinterop2#dragleave)
* [ICoreWebView2CompositionControllerInterop2.DragOver Method](/microsoft-edge/webview2/reference/winrt/interop/icorewebview2compositioncontrollerinterop2#dragover)
* [ICoreWebView2CompositionControllerInterop2.Drop Method](/microsoft-edge/webview2/reference/winrt/interop/icorewebview2compositioncontrollerinterop2#drop)
```

File: microsoft-edge/webview2/concepts/overview-features-apis.md [hunk 4]
```diff
@@ -1482,6 +1482,33 @@ Spatial input (mouse, touch, pen) is received by the application and must be sen
...
+
+##### [Win32/C++](#tab/win32cpp)
+
+* [ICoreWebView2CompositionController3::DragEnter method](/microsoft-edge/webview2/reference/win32/icorewebview2compositioncontroller3?view=webview2-1.0.1370.28&preserve-view=true#dragenter)
+* [ICoreWebView2CompositionController3::DragLeave method](/microsoft-edge/webview2/reference/win32/icorewebview2compositioncontroller3?view=webview2-1.0.1370.28&preserve-view=true#dragleave)
+* [ICoreWebView2CompositionController3::DragOver method](/microsoft-edge/webview2/reference/win32/icorewebview2compositioncontroller3?view=webview2-1.0.1370.28&preserve-view=true#dragover)
+* [ICoreWebView2CompositionController3::Drop method](/microsoft-edge/webview2/reference/win32/icorewebview2compositioncontroller3?view=webview2-1.0.1370.28&preserve-view=true#drop)
```
Comment (mikehoffms) [inline microsoft-edge/webview2/concepts/overview-features-apis.md:1508]:
```suggestion
* [ICoreWebView2CompositionController3::DragEnter method](/microsoft-edge/webview2/reference/win32/icorewebview2compositioncontroller3#dragenter)
* [ICoreWebView2CompositionController3::DragLeave method](/microsoft-edge/webview2/reference/win32/icorewebview2compositioncontroller3#dragleave)
* [ICoreWebView2CompositionController3::DragOver method](/microsoft-edge/webview2/reference/win32/icorewebview2compositioncontroller3#dragover)
* [ICoreWebView2CompositionController3::Drop method](/microsoft-edge/webview2/reference/win32/icorewebview2compositioncontroller3#drop)
```

**Review by mikehoffms — APPROVED** [review]

**Review by Reezaali — APPROVED** [review]
Approving to unblock the merge.

**Pull request merged**